schema/testing.proto

```diff
@@ -55,123 +92,13 @@ service Testing {
   //
   // In cases where a test involves registering a final answer at the
   // end of the test, this method provides the means to do so.
-  rpc RegisterTest(RegisterTestRequest) returns (google.protobuf.Empty) {
+  rpc CheckAnswer(CheckAnswerRequest) returns (CheckAnswerResponse) {
```

Comment:
The noun here is "Test". `CheckTest`, perhaps?

Reply:
Agreed.